### ISSUE TITLE
Layout improvements for discussion

### ISSUE THREAD

**Issue by leucoryx** [rdmorganiser/rdmo#254]
1.    Hilfetext zu Beschreibung und Katalog wie z.B. “Der Fragenkatalog der für dieses Projekt verwendet wird.“ sollte oben drüber stehen, nicht unten drunter, konsistenz zu wie es auch im Fragenkatalog bei Fragen gehandhabt wird.
2.    Projekt anlegen. Titel kann als value project/title bei Ansichten ausgelesen werden. Wenn auch eine Frage mit attribut project/title angelegt wird, wird der value dort aber nicht übernommen. Wieso? Gäbe es diese Möglichkeit? Wie ist das technisch umgesetzt?
3.    “Weiteren” Stellvertreter hinzufügen, „Weiteres Schlagwort hinzufügen“ etc., um es klarer zu machen
4.    „Fragen beantworten“ zu wenig prominent, dafür, dass es die erste Aktion des Nutzers sein sollte. 
5.    Aufgaben leer irritiert beim ersten Aufruf. Unklar, was das soll. Erwartung, dass man selber Aufgaben anlegen kann -> Text ändern in "Keine aktiven Aufgaben gefunden. Bitte beantworten Sie zunächst die Fragen im Fragenkatalog, indem Sie im Menü „Fragen beantworten“ klicken. Aus Ihrer Beantwortung der Fragen werden notwendige Aufgaben generiert"
6.    Ansichten sollte lieber “Formulare” oder ähnliches heißen. Wording “Ansicht” ist unklar / Unklar, dass Fragen/Antworten die Ansicht befüllen.
7.    „Katalog“ ist unklar. Evtl. ändern in „Fragenkatalog“
8.    “Snapshot” ist unklar, ist das eine Vorschau/ein Stand? kurze Erklärungstexte
9.    “Mitgliedschaften” Formulierung unpassend -> Mitglieder?
10. Katalogauswahl per Dropdownmenü
11. Pflichtfelder und optionale Felder vorgeben können
12. Filtern von Fragen auf aktueller Seite ermöglichen und je nach Antwortauswahl Fragen dazu ein-/ausblenden.
Nice-to-have
1.    “Sichern” und “Sichern und fortfahren” Wording ist unklar → Speichern und Speichern und fortfahren
2.    Beschreibung → Hilfetext, was hier rein soll. Sollte dort tatsächlich nur optional stehen?
3.    Beim Menü rechts “Projektinformationen bearbeiten” aber “Projekt Aufgaben” und “Projekt Ansichten”. Konsistenter und eingängiger wäre “Projektaufgaben” “Projektansichten” etc.
4.    Bei Select File sollte vorausgewählt sein, dass nur xml files gehen
5.    “Zur nächsten Seite springen” müsste es heißen, zur nächsten Frage würde implizieren, dass man auf der aktuellen Seite bleibt und zur nächsten Frage springt   (im Englischen ist es richtig)
6.    Fortschrittsbalken beim beantworten der Fragen durch zahlen verstärken (seite x von y)
7.    Kalender sollte mit Mo anfangen, nicht mit Sonntag

**Comment by YvanaGlasenapp** [comment]
I agree with the suggested improvements; especially number 4, a more prominent start-button.

**Comment by jochenklar** [comment]
As of October 23:

1. already implemented
2. this is not possible, we could use something else than `project/title`, but I would rather advise people not to use that attribute.
3. The translated buttons will be updated in `2.1.0`, but with less text, since it causes trouble with translations. It will be only "+ Stellvertreter".
4. We moved this to the top of the sidebar, but maybe it will be changed again in an overhaul of the project page.
5. The help text was already adjusted.
6. The term view/Ansicht is now pretty established in the community. We might combine "Antworten anzeigen" with the views in the future though.
7. **We should discuss this in the community.** Separate Issue: #781 this.**
8. The help text was already adjusted.
9. **Again, needs to be discussed. The help text was already adjusted.** Separate Issue: #782.
10. Already implemented as an external template. https://github.com/jochenklar/rdmo.jochenklar.de/blob/master/rdmo_jochenklar/templates/projects/project_detail_header_catalog.html
11. Already implemented.
12. If this is part of the management, then it is already implemented. For the interview, I don't see the point.
13. Already implemented.
14. Already implemented.
15. Will be fixed in `2.1.0`.
16. Will be fixed in `2.1.0`.
17. Not relevant anymore, because of the change to the buttons.
18. A "new" progress bar will be part of `2.1.0`
19. Already implemented.

Regarding @YvanaGlasenapp idea we would need to discuss that with the community as well. Separate Issue: #783.
